src/springfield/fields.py

```diff
@@ -339,6 +340,20 @@ class UrlField(StringField):
     """
     :class:`Field` with a URL value
     """
+    def adapt(self, value):
+        """
+        Validate that the `value` has a valid URL format containing
+        a scheme and network location using urlparse.
+
+        Supported schemes include `http`, `https` and `ftp`.
```

Comment:
I can just verify we found something for that field if you prefer to leave it more open.